homeassistant/components/schluter/__init__.py

```diff
@@ -0,0 +1,84 @@
+"""The Schluter DITRA-HEAT integration."""
+from datetime import timedelta
+import logging
+
+from requests import RequestException, Session
+from schluter.api import Api
+from schluter.authenticator import AuthenticationState, Authenticator
+import voluptuous as vol
+
+from homeassistant.const import CONF_PASSWORD, CONF_TIMEOUT, CONF_USERNAME
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+DATA_SCHLUTER_SESSION = "schluter_session"
+DATA_SCHLUTER_API = "schluter_api"
+DEFAULT_TIMEOUT = 10
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=5)
+SCHLUTER_CONFIG_FILE = ".schluter.conf"
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_USERNAME): cv.string,
+                vol.Required(CONF_PASSWORD): cv.string,
+                vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+
+async def async_setup_schluter(hass, config, api, authenticator):
```

Comment:
You have marked this as an async function but you are calling sync functions. Please not mark this as async.

Reply:
Async has been removed from this function (and function name modified)